manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml

```diff
@@ -110,6 +110,9 @@ spec:
             - containerPort: {{ .Values.livenessProbe.port }}
               name: healthz
               protocol: TCP
+            - containerPort: {{ .Values.windows.metricsAddr }}
```

Comment:
The `metricsAddr` helm value is a string of the format `:<port number>`

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/181f3bd87a2159df5d952e390a9e13068b753e36/charts/secrets-store-csi-driver/values.yaml#L67

So this will error out. The format for metricsAddr is also part of the flag config and we won't be able to change that. Could we parse the value using a helm template function?

Reply:
Will give it a thought on how I can solve it.

Reply:
@NissesSenap gental ping. Please let us know if you need help with the PR

Reply:
Yeah sorry about this life and other tasks came in the way. 

I gave it a few tries to write a good tpl function that would cover the use case as but was unable to do so.
If you have a pointer @nilekhc that would be great.

Reply:
@NissesSenap you can use `trimPrefix` function here.

`{{ trimPrefix ":" .Values.windows.metricsAddr }}`

Reply:
```suggestion
            - containerPort: {{ trimPrefix ":" .Values.windows.metricsAddr }}
```

Reply:
That did the trick, thanks @nilekhc!
Earlier I tried to do interesting tpl functions with fmt stuff. But of course a simple trimPrefix solved it.
Some times you just go to deep in to a single thought...